aten/src/ATen/quantized/Quantizer.cpp

```diff
@@ -171,30 +187,109 @@ T quantize_val(double scale, int64_t zero_point, float value) {
   int64_t qvalue;
   constexpr int64_t qmin = std::numeric_limits<typename T::underlying>::min();
   constexpr int64_t qmax = std::numeric_limits<typename T::underlying>::max();
-  checkZeroPoint<typename T::underlying>("quantize_val", zero_point);
-  qvalue = static_cast<int64_t>(std::nearbyint(value / scale + zero_point));
+  qvalue = static_cast<int64_t>(Round(value / scale + zero_point));
   qvalue = std::max<int64_t>(qvalue, qmin);
   qvalue = std::min<int64_t>(qvalue, qmax);
   return static_cast<T>(qvalue);
 }
 
 template <typename T, int precision>
 void quantize_vec(double scale, int64_t zero_point, const float *src, T *dst, size_t count) {
+  checkZeroPoint<typename T::underlying>("quantize_val", zero_point);
   for (int64_t i = 0; i < count; ++i) {
     dst[i] = quantize_val<T>(scale, zero_point, src[i]);
   }
 }
 
+#ifdef GEMMLOWP_NEON
```

Comment:
looks like the only header you need is third_party/gemmlowp/gemmlowp/internal/detect_platform.h . And maybe even that better to pull out if we don't depend on gemmlowp otherwise

Reply:
Removed this and add an explicit `__ARM_NEON__` define to guard this.